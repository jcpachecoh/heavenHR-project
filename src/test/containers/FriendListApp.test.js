import React from 'react';
import FriendListApp from '../../containers/FriendListApp';
import * as TestUtils from 'react-addons-test-utils';
import {combineReducers, createStore} from 'redux';
import { Provider } from 'react-redux';
import * as reducers from '../../reducers';

const reducer = combineReducers(reducers);
const store = createStore(reducer);

describe('Testing <FriendListApp /> component', () => {
    it('Should render FriendListApp component ', () => {
    
        expect(TestUtils.createRenderer().render(
            <Provider store={store}>
                <FriendListApp />
            </Provider>
          )).toMatchSnapshot()
    });
});
