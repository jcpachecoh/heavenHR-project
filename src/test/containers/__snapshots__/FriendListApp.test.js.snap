// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <FriendListApp /> component Should render FriendListApp component  1`] = `<Connect(FriendListApp) />`;
